docs/backtesting.md

```diff
@@ -46,14 +14,13 @@ real data. This is what we call
 Backtesting will use the crypto-currencies (pairs) from your config file
 and load ticker data from `user_data/data/<exchange>` by default.
 If no data is available for the exchange / pair / ticker interval combination, backtesting will
-ask you to download them first using `freqtrade download-data`. 
-For details on downloading, please refer to the [relevant section](#Getting-data-for-backtesting-and-hyperopt) in the documentation.
+ask you to download them first using `freqtrade download-data`.
+For details on downloading, please refer to the [Downloading data](data-download.md) section in the documentation.
```

Comment:
The section named "Data download" in its file. Maybe "Data downloading"/"Downloading data" is even better. But should be same in all places.